src/transformers/utils/quantization_config.py

```diff
@@ -386,13 +389,15 @@ def __init__(
         self.module_name_preceding_first_block = module_name_preceding_first_block
         self.batch_size = batch_size
         self.pad_token_id = pad_token_id
-        self.disable_exllama = disable_exllama
+        self.use_exllama = use_exllama
         self.max_input_length = max_input_length
+        self.use_exllama_v2 = use_exllama_v2
+        self.disable_exllama = kwargs.get("disable_exllama", None)
```

Comment:
We need to have arguments verification here to make sure that `disable_exllama` is compatible with the other exllama arguments